server/services/token.service.js

```diff
@@ -0,0 +1,35 @@
+import jwt from "jsonwebtoken";
+import ApiError from "../error/api.error.js";
+
+export default class TokenService {
+  generateTokens(payload) {
+    try {
+      const accessToken = jwt.sign(payload, process.env.JWT_ACCESS_SECRET, {
+        expiresIn: "30m",
+      });
+      const refreshToken = jwt.sign(payload, process.env.JWT_REFRESH_SECRET, {
+        expiresIn: "30d",
+      });
+
+      return { accessToken, refreshToken };
+    } catch (e) {
+      throw ApiError.internal("Failed generate tokens!");
+    }
+  }
+
+  validateAccessToken(token) {
+    try {
+      return jwt.verify(token, process.env.JWT_ACCESS_SECRET);
+    } catch (e) {
+      return null;
+    }
+  }
+
```

Comment:
Not sure that you need two different secrets and two different methods but it's OK
I mean, you could use one secret and one method for both refresh and access tokens. But it's OK